python/pyspark/__init__.py

```diff
@@ -51,6 +51,26 @@
 # for back compatibility
 from pyspark.sql import SQLContext, HiveContext, SchemaRDD, Row
 
+
+def since(version):
```

Comment:
This should belong to SPARK-10373. We should have a separate PR for it and then update this PR. ping @davies